drivers/serial/Kconfig.xlnx

```diff
@@ -7,5 +7,6 @@ config UART_XLNX_PS
 	bool "Xilinx serial driver for MPSoC"
 	depends on SOC_XILINX_ZYNQMP
 	select SERIAL_HAS_DRIVER
+    select SERIAL_SUPPORT_INTERRUPT
```

Comment:
Please use TAB here (changes should align with what the rest of the file is doing)